### PR TITLE
fix(pkg): delete unused var or const

### PR DESCRIPTION
**Pull request by hwdef**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

delete unused var or const in pkg

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: pkg/util/ebtables/ebtables.go [hunk 1]
```diff
@@ -62,8 +62,6 @@ const (
 	opFlushChain  operation = "-F"
 	opDeleteChain operation = "-X"
 	opListChain   operation = "-L"
-	opAppendRule  operation = "-A"
```
Comment (thockin) [inline pkg/util/ebtables/ebtables.go]: These were added for completeness.  Lets leave these ones.

File: pkg/util/iptables/iptables.go [hunk 2]
```diff
@@ -564,7 +564,6 @@ const (
 	opFlushChain  operation = "-F"
 	opDeleteChain operation = "-X"
 	opListChain   operation = "-L"
-	opAppendRule  operation = "-A"
```
Comment (thockin) [inline pkg/util/iptables/iptables.go]: Same with this.  It's documenting an API

**Comment by hwdef** [comment]
@thockin 
I fix them.
Thanks for your suggestions.

**Comment by hwdef** [comment]
@thockin 
Cloud you please add an approved label for this PR?
Thanks

**Review by thockin — COMMENTED** [review]
Thanks!

/lgtm
/approve

**Pull request merged**